tools/build_defs/framework.bzl

```diff
@@ -401,35 +399,33 @@ def _define_outputs(ctx, attrs, lib_name):
 
     _check_file_name(lib_name, "Library name")
 
-    installdir = ctx.actions.declare_directory(lib_name)
```

Comment:
Let's keep installdir as is (in my PR I am making a copy)
* Please read the comment to the PR first, with more detailed explanation